src/sentry/nodestore/base.py

```diff
@@ -237,7 +237,8 @@ def set_subkeys(self, id, data, ttl=None):
         >>> nodestore.get('key1', subkey='reprocessing')
         {'foo': 'bam'}
         """
-        with sentry_sdk.start_span(op="nodestore.set_subkeys") as span:
+        with sentry_sdk.start_span(op="nodestore", name="nodestore.set_subkeys") as span:
+            sentry_sdk.set_tag("nodestore.set_subkeys", True)
```

Comment:
If we can't get Performance to help us slice this, at least we can use a tag.

FYI I've noticed this being executed when an event has attachments.